ztp/gitops-subscriptions/argocd/CoreReduction.md

```diff
@@ -0,0 +1,37 @@
+# Core Reduction with Workload Partitioning and Performance Profile
```

Comment:
Should we mention 'core reduction' at all here? I think we've historically used that term to mean the effort of reducing the CPU usage for all platform services, workload partitioning is a feature that can be used with out an intent to reduce actual CPU usage. Maybe we just call this `# Workload Partitioning` to keep it simple on the intent of this section?

Reply:
Yeah, maybe we say the whole section is workload partitioning, and we make a change of `Part 1 - Configure Node Processes` and `Part 2 - Configure Cluster Processes`, something to that effect.